addons/nuxeo-retention/nuxeo-retention-web-ui/src/main/resources/web/nuxeo.war/ui/document/retentionrule/nuxeo-retentionrule-create-layout.html

```diff
@@ -0,0 +1,197 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this retentionrule except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-retentionrule-create-layout`
+@group Nuxeo UI
+@element nuxeo-retentionrule-create-layout
+-->
+<dom-module id="nuxeo-retentionrule-create-layout">
+
+  <template>
+    <style include="iron-flex iron-flex-alignment nuxeo-styles">
+      *[role=widget] {
+        margin-bottom: 8px;
+      }
+    </style>
+
+    <nuxeo-input role="widget"
+                 label="[[i18n('retention.rule.label.ruleName')]]"
+                 name="title"
+                 value="{{document.properties.dc:title}}"
+                 autofocus required>
+    </nuxeo-input>
+
+    <nuxeo-input role="widget"
+                 label="[[i18n('retention.rule.label.ruleDescription')]]"
+                 name="description"
+                 value="{{document.properties.dc:description}}">
+    </nuxeo-input>
+
+    <paper-checkbox role="widget" name="disabled" checked="{{document.properties.retention_rule:enabled}}">
+      [[i18n('retention.rule.label.enabled')]]
+    </paper-checkbox>
+
+    <nuxeo-card heading="[[i18n('retention.rule.label.applicationPolicy.heading')]]">
+      <nuxeo-select id="applicationPolicyDropdown" selected="{{document.properties.retention_rule:applicationPolicy}}"
+        attr-for-selected="key"
+        role="widget"
+        disabled
+        placeholder="[[i18n('retention.rule.label.applicationPolicy.placehodler')]]"
+        label="[[i18n('retention.rule.label.applicationPolicy.description')]]" required>
+        <paper-item key="auto">[[i18n('retention.rule.label.applicationPolicy.auto')]]</paper-item>
+        <paper-item key="manual">[[i18n('retention.rule.label.applicationPolicy.manual')]]</paper-item>
+      </nuxeo-select>
+      <dom-if if="[[_isAuto(document.properties.retention_rule:applicationPolicy)]]">
+        <template>
+          <nuxeo-input role="widget"
+                       label="[[i18n('retention.rule.label.applicationPolicy.auto.description')]]"
+                       placeholder="Document.getType() == 'File'"
```

Comment:
Is this what we want as `placeholder`?

Reply:
No idea. @Gabez0r / @guirenard ?

Reply:
Or @nmpcunha ?

Reply:
I'd say so, since this is bound to a field holding an EL expression to be executed by the retention policy. It's an example.